server/pages/index.jade

```diff
@@ -55,6 +55,9 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 			else
 				link(rel='stylesheet', href=urls['style.css'])
 
+		if oauth
+			link(rel='stylesheet', href='/calypso/css/' + oauth + '.css')
```

Comment:
I don't think we'll have a large amount of customization for each oauth client. We could simply bundle all those in the same file imo